lms/templates/courseware/program_marketing.html

```diff
@@ -1,3 +1,407 @@
 <%page expression_filter="h"/>
+<%inherit file="../main.html"/>
```

Comment:
All new templates should be Django templates: http://open-edx-proposals.readthedocs.io/en/latest/oep-0011.html#technology-selection. Death to Mako!